src/variant.ts

```diff
@@ -203,7 +214,7 @@ const jsonifyVariant = (variant: VariantNotation): { [key: string]: string } =>
     return json;
 };
 
-const stringifyVariant = (variant: VariantNotation): string => {
+const stringifyVariant = (variant: VariantNotation, newFusion = false): string => {
```

Comment:
Does 'newFusion' just control whether the output is in the new style, or is it required if the input is in the new style as well? If both, would it be helpful to have two params - one for returning the new style of output, one for expecting the new style of input?

Reply:
Is it just the check for two '::' matches that's used to determine whether the input is the new style?

Reply:
It's just for the output.
newFusion = false will return the legacy displayName like "(ZNF532,NUTM1):fusion(p.?4,e.2)";
newFusion = true will return the new nomenclature displayName like "ZNF532:p.??_?4::NUTM1:e.2_?"